tools/nni_cmd/nnictl_utils.py

```diff
@@ -328,13 +328,18 @@ def log_trial(args):
         print_error('Restful server is not running...')
         exit(1)
     if args.id:
```

Comment:
what's the effect of args.id here?

Reply:
@SparkSnail 
To check whether experiment id exists. If not, print all possible trial id list & log_path.